python/cm/adcm_config.py

```diff
@@ -995,10 +1008,13 @@ def set_object_config(obj, keys, value):
     # if config_is_ro(obj, keys, pconf.limits):
     #    msg = 'config key {} of {} is read only'
     #    err('CONFIG_VALUE_ERROR', msg.format(key, ref))
-    replace_object_config(obj, key, subkey, value)
+    replace_object_config(obj, key, subkey, value, pconf)
     if pconf.type == 'file':
         save_file_type(obj, key, subkey, value)
-    log.info('update %s config %s/%s to "%s"', obj_ref(obj), key, subkey, value)
+    log_value = value
+    if pconf.type in ('password', 'secrettext'):
```

Comment:
Maybe store ('password', 'secrettext') as a const somewhere?
like `SECURE_PARAM_TYPES`
to reuse them anywhere it's required

Reply:
done